### PR TITLE
Icons: adding FontAwesome's linkedin-square icon

### PR DESCRIPTION
**Pull request by talbs**

### CODE REVIEW

**Comment by talbs** [comment]
@clrux, mind reviewing this quick addition?

- - -

FYI, @frrrances

File: _src/pattern-library/images/edx-svg/svgdefs.svg [hunk 1]
```diff
@@ -1,5 +1,8 @@
-<svg display="none" width="0" height="0" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<svg style="position: absolute; width: 0; height: 0;" width="0" height="0" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (clrux) [inline _src/pattern-library/images/edx-svg/svgdefs.svg]: Was this done automatically?

Comment (talbs) [inline _src/pattern-library/images/edx-svg/svgdefs.svg]: Good catch, yeah, I didn't edit this.

**Comment by frrrances** [comment]
ouch what a diff! changes look good, though i don't really understand the svg diff you guys commented on... you're going to leave it with the auto output and not edit that file, right? Otherwise, :+1:

**Comment by clrux** [comment]
:+1:

**Pull request merged**